enclave-runner/src/loader.rs

```diff
@@ -200,6 +205,10 @@ impl<'a> EnclaveBuilder<'a> {
         self
     }
 
+    pub fn user_call_ext_impl(&mut self, e: Box<UsercallExtension>) {
```

Comment:
Please add
```
impl<T: UsercallExtension> Into<Box<UsercallExtension>> for T { ... }
```
And change this function so it takes `T: Into<Box<UsercallExtension>>`

Reply:
Thanks. Should I do something similar for Box<SyncStream> too?